src/app/components/client/ProgressCard.tsx

```diff
@@ -0,0 +1,110 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ "use client";
+
+ import { CSSProperties, useState } from "react";
+ import { QuestionMarkCircle } from "../server/Icons";
+ import styles from "./ProgressCard.module.scss";
+ import ExploringLaptopPlus from "./assets/exploring-laptop-check.png";
+ import ExploringLaptopMinus from "./assets/exploring-laptop-minus.png";
+ import SparklingCheck from "./assets/sparkling-check.png";
+ import Image from "next/image";
+import { Modal } from "./Modal";
+ 
+ export type Props = {
+   resolvedByYou: number;
+   autoRemoved: number;
+   totalNumExposures: number;
+ };
+ 
+ function PercentageComplete(props: Props) {
+   const percentageCompleteNum =
```

Comment:
Suggestion (blocking): The calculation for the percentage has two potential problems as already pointed by @pdehaan.

- We need to guard against division by zero.
![progress-card-infinity](https://github.com/mozilla/blurts-server/assets/13835474/27dbdfce-6032-4dbb-8e17-3f6115581408)

- We should round the values for `complete` and `in progress`
![progress-card-round-values](https://github.com/mozilla/blurts-server/assets/13835474/7fa61bc8-68fa-42c0-ad03-7471c0f9aa9d)

Reply:
nit: Not sure if we'd need a better "empty" state UI.
The Storybook mockup above shows:
> Resolved by you: 0
> Auto-removed: 0
> 0% complete
> 100% in progress

Except there is nothing in progress if there are no exposures.